test/Model/Fulfilment/OrderNoteTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Sdk\Test\Model\Fulfilment;
+
+use MyParcelNL\Sdk\src\Model\Fulfilment\OrderNote;
+use MyParcelNL\Sdk\Test\Bootstrap\TestCase;
+
+class OrderNoteTest extends TestCase
+{
+    public function testToArray(): void
+    {
+        $orderNote = (new OrderNote())
+            ->setOrderUuid('uuid')
+            ->setAuthor('customer')
+            ->setNote('Test');
+
+        $this->assertEquals('uuid', $orderNote->getOrderUuid());
+        $this->assertEquals('customer', $orderNote->getAuthor());
+        $this->assertEquals('Test', $orderNote->getNote());
+
+        $this->assertEquals((object)[
+            'author' => 'customer',
+            'note'   => 'Test',
+        ], $orderNote->toApiObject());
+    }
+
+    /**
+     * @param  array $data
+     *
+     * @throws \Exception
+     * @dataProvider bunchOfNotes
+     */
+    public function testValidator(array $data) {
+        $this->expectExceptionMessage($data['expectedError']);
+        $note = new OrderNote($data['orderNote']);
+        $note->validate();
+    }
+
+    public function bunchOfNotes(): array
```

Comment:
volgens phpunit best practices(?) moet deze functie `provide<...>` of `<...>Provider` oid heten. dan weet je ook meteen dat het om een data provider method gaat. (Ik weet dat dit op de meeste andere plekken ook niet klopt)